src/controller/adverse.ts

```diff
@@ -22,7 +23,13 @@ export const AddAdverse = (req: Request, res: Response, next: NextFunction) => {
     const name = req.body.name;
     const pre_notice_date = req.body.pre_notice_date;
     const post_notice_date = req.body.post_notice_date;
```

Comment:
```suggestion
    const { name, pre_notice_date, post_notice_date } = req.body;
```
you can use the above line instead of individualy doing it